test/worldwide/region_test.rb

```diff
@@ -15,6 +15,8 @@ class RegionTest < ActiveSupport::TestCase
       assert_equal "ZZ", region.iso_code
       assert_nil region.legacy_code
       assert_nil region.legacy_name
+      assert_empty(region.format)
+      assert_empty(region.format_extended)
```

Comment:
<img width="599" alt="image" src="https://github.com/Shopify/worldwide/assets/33674553/99b360cc-44ed-4e6c-8eb9-bc3028220815">

Reply:
Interesting... I don't love that rubocop rule but it's better than nothing